doc/contributing.md

```diff
@@ -363,7 +363,7 @@ General guidelines for making a good gallery plot:
 * Use a sample dataset from `pygmt.datasets` if you need to plot data. If a suitable
   dataset isn't available, open an issue requesting one and we'll work together to add
   it.
-* Add comments to explain things are aren't obvious from reading the code. Good: *Use a
+* Add comments to explain things aren't obvious from reading the code. Good: *Use a
```

Comment:
Missing `that`?
```suggestion
* Add comments to explain things that aren't obvious from reading the code. Good: *Use a
```

Reply:
I think both versions work but with 'that' it sounds better. Thanks!